packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -166,10 +166,12 @@ function getFilteredGlobalObjectShapeDescriptors(endowments: object): PropertyDe
 }
 
 export function getResolvedShapeDescriptors(
-    globalObjectShape: object,
+    globalObjectShape?: object,
     endowments?: object
 ): PropertyDescriptorMap {
-    const globalObjectShapeDescriptors = getFilteredGlobalObjectShapeDescriptors(globalObjectShape);
+    const globalObjectShapeDescriptors = getFilteredGlobalObjectShapeDescriptors(
```

Comment:
we should probably not even call the filter if the value is not defined

Reply:
This was from the other commit. We should discuss what we want to do if shape object is missing? Throw, or use an empty object?